tools/pontoon/adding_new_short_term_project.md

```diff
@@ -0,0 +1,43 @@
+# Adding a new short-term project on Pontoon
+
+Short-term projects are things like newsletters, marketing campains, surveys... They do not have a repository, and data is instead stored only in Pontoon's database.
```

Comment:
nits:
* `...` should be either `etc.` or `…` (single Unicode character). 
* Use ’ instead of ' (apostrophe instead of straight single quote): Pontoon's -> Pontoon’s. This also later in the doc.